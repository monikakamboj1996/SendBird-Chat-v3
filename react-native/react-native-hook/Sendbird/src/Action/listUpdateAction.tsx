export const ListUpdate = list => {
  return {
    type: 'List_Update',
    list: list,
  };
};
