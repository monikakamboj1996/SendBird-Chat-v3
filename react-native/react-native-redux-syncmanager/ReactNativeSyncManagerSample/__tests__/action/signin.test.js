import 'react-native';
import React from 'react';
import Action from '../../src/action/signin';

describe('action/authenticate', () => {
  it('authenticate', () => {
    const result = Action.authenticate();
    expect(result).toMatchSnapshot();
  });

  it('succeed', () => {
    const result = Action.succeed({ nickname: 'test' });
    expect(result).toMatchSnapshot();
  });

  it('fail', () => {
    const result = Action.fail(new Error('fail'));
    expect(result).toMatchSnapshot();
  });
});
