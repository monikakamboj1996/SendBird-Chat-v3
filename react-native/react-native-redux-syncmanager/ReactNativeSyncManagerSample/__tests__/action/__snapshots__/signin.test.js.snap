// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action/authenticate authenticate 1`] = `
Object {
  "type": "authenticate",
}
`;

exports[`action/authenticate fail 1`] = `
Object {
  "payload": Object {
    "err": [Error: fail],
  },
  "type": "signin-failed",
}
`;

exports[`action/authenticate succeed 1`] = `
Object {
  "payload": Object {
    "user": Object {
      "nickname": "test",
    },
  },
  "type": "signin-success",
}
`;
